tests/test_diffraction_objects.py

```diff
@@ -212,7 +212,7 @@ def test_init_invalid_xtype():
             },
             {"xtype": "tth", "yarray": np.array([4.1, 5.1, 6.1, 7.1, 8.1, 9.1])},
         ),
-        (  # C2: Same length x-arrays with exact x-value match
+        (  # C2: Same x-arrays, x-value matches at tth=60, expect y-array to divide by 10
```

Comment:
If i understand correctly, we compare the yarray_1 of 60 against yarray_2 of 6 (located at xarray of 60) and we expect the y-array to e divided by 10 based on this ratio?

Just comment - since both xarray and yarray use 60 for the first input, it's a bit hard to follow

Reply:
Yes! It finds the closest index in each x-array and scales based on that. In this case the scaling factor is 60/6=10 and the y-array for the original diffraction object is divided by 10